django_project/certification/views/certificate.py

```diff
@@ -221,6 +224,14 @@ def generate_pdf(
         pathname, project, course, attendee, certificate, current_site):
     """Create the PDF object, using the response object as its file."""
 
+    # Register new font
+    try:
+        font_folder = os.path.join(settings.STATIC_ROOT, 'fonts/NotoSans-hinted')
+        ttf_file = os.path.join(font_folder, 'NotoSans-Bold.ttf')
+        pdfmetrics.registerFont(TTFont('Noto-bold', ttf_file))
+    except:
```

Comment:
__[Flake8]__

[E722] do not use bare except'


<!-- Comment by [Sider](https://sider.review) -->